cmd/postcli/main.go

```diff
@@ -198,3 +211,27 @@ func saveKey(key ed25519.PrivateKey) error {
 	}
 	return nil
 }
+
+func cmdVerifyPos(opts config.InitOpts, fraction float64) int {
+	params := postrs.TranslateScryptParams(opts.Scrypt.N, opts.Scrypt.R, opts.Scrypt.P)
+	verifyOpts := []postrs.VerifyPosOptionsFunc{
+		postrs.WithFraction(fraction),
+		postrs.FromFile(uint32(opts.FromFileIdx)),
+	}
+	if opts.ToFileIdx != nil {
+		verifyOpts = append(verifyOpts, postrs.ToFile(uint32(*opts.ToFileIdx)))
+	}
+
+	err := postrs.VerifyPos(opts.DataDir, params, verifyOpts...)
```

Comment:
Why not just return the error and if `err != nil` then `log.Fatal("failed to verify: %s", err)` ?

Reply:
Didn't think of log.Fatal. Sure I can use it, but I still want to differentiate between "verification failure" and "verification successful but pos is invalid".